module/applications/api/document-sheet.mjs

```diff
@@ -1,30 +1,13 @@
-const { DocumentSheetV2, HandlebarsApplicationMixin } = foundry.applications.api;
+import V2Mixin from "./v2-mixin.mjs";
+
+const { DocumentSheetV2 } = foundry.applications.api;
 
 /**
  * Base document sheet from which all document-based application should be based.
  */
-export default class DocumentSheet5e extends HandlebarsApplicationMixin(DocumentSheetV2) {
+export default class DocumentSheet5e extends V2Mixin(DocumentSheetV2) {
```

Comment:
We unfortunately have overloaded the V2 term so it's not clear which 'V2' this refers to from just the name. Maybe we can call it something like `ApplicationV2Mixin`?